### PR TITLE
Add a section explaining WEAVE_MTU for fastdp

### LINKED ISSUE #2198: Documentation section about MTU configuration is missing
```
Ensure that WEAVE_MTU fits with the MTU of the intermediate network (see below)
```

There's nothing below.

### PR DESCRIPTION
**Pull request by bboreham**
Fixes #2198

### CODE REVIEW

File: site/using-weave/fastdp.md [hunk 1]
```diff
@@ -42,6 +42,23 @@ Where fastdp indicates that fast datapath is being used on a connection. If fast
     $ weave status connections
     <- 192.168.122.25:54782  established sleeve 8a:50:4c:23:11:ae(ubuntu1204)
 
+###Packet size (MTU)
+
+The Maximum Transmission Unit, or MTU, is the technical term for the
+limit on how big a single packet can be on the network. Weave Net
```
Comment (rade) [inline site/using-weave/fastdp.md]: "Weave Net in fastdp mode"

File: site/using-weave/fastdp.md [hunk 2]
```diff
@@ -42,6 +42,23 @@ Where fastdp indicates that fast datapath is being used on a connection. If fast
...
+defaults to 1410 bytes, but you can set a smaller size if your
+underlying network has a tighter limit, or set a larger size for
+better performance.
+
+The underlying network must be able to deliver packets of the size
+specified plus overheads of around 50 bytes.  This requirement applies
+to every path between peers.
```
Comment (rade) [inline site/using-weave/fastdp.md]: I'd emphasise "*every path*".

File: site/using-weave/fastdp.md [hunk 3]
```diff
@@ -42,6 +42,23 @@ Where fastdp indicates that fast datapath is being used on a connection. If fast
...
+
+To specify a different MTU, before launching Weave Net set the
+environment variable `WEAVE_MTU`.  For example:
+
+    $ WEAVE_MTU=1350 weave launch host2 host3
```
Comment (rade) [inline site/using-weave/fastdp.md]: A typical "jumbo frame" MTU might be a better example here.

**Comment by rade** [comment]
cc @abuehrle

File: site/using-weave/fastdp.md (hunk 1 above)
Comment (bboreham) [inline site/using-weave/fastdp.md]: The same limit applies in 'sleeve' mode.

Comment (rade) [inline site/using-weave/fastdp.md]: since when? We used to default to 64k.

Comment (bboreham) [inline site/using-weave/fastdp.md]: May last year. https://github.com/weaveworks/weave/commit/431197042d14570cb2e1e8e1fb70418e2142b4e4

**Comment by bboreham** [comment]
Updated.

**Comment by abuehrle** [comment]
LGTM  but I'm not sure about linking to an article in the blog.  At some point we should take that information out and include in the guide as a topic, since it is important information.

**Pull request merged**